src/public_parameters/mod.rs

```diff
@@ -409,957 +48,26 @@ where
     }
 
     fn read_from_path<P: AsRef<Path>>(path: P) -> Result<Self, Error> {
```

Comment:
This impl block was restored to the previous behavior on master, it should be the same as https://github.com/lurk-lab/lurk-rs/blob/master/src/public_parameters/mod.rs#L400

Reply:
I must have messed up the rebase here, I reverted it to what's in master.